starter-code/public/scripts/app.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+function Project (entry) {
+  this.projCategory = entry.projCategory;
+  this.projTitle = entry.projTitle;
+  this.projSrc = entry.projSrc;
+  this.projUrl = entry.projUrl;
+  this.projSummary = entry.projSummary;
+}
+
+Project.all = [];
+
+Project.prototype.toHtml = function() {
+  let template  = Handlebars.compile($('#project-template').text());
+  return template(this);
+};
+// var projectSource = $('#project-template').html();
+// var handleBarsTemplate = Handlebars.compile(projectSource);
+
+// Pass our data to the template
+// var theCompiledHtml = handleBarsTemplate(rawData);
+
+// Add the compiled html to the page
+// $('#projects').append(theCompiledHtml);
+Project.loadAll = function(rawData) {
+  rawData.forEach(function(ele) {
+    Project.all.push(new Project(ele));
+  })
+}
+//initiate json file.
+Project.fetchAll = function() {
+  if (localStorage.rawData) {
+    Project.loadAll(JSON.parse(localStorage.rawData));
+    var initIndexPage = function() {
```

Comment:
You declare this function but, you arent calling it. I would compare your code to lab 8 i think to see the difference.